data_architect_misc/data_transformer/transform_functions/common_comp_harm_transform_functions.py

```diff
@@ -64,21 +65,20 @@ def create_new_dataframe_from_input_CSV_files(
             the input files provided as paramter to this
             function.
         """
+        list_of_file_path_and_names = glob(folder_name+'/*')
```

Comment:
When you join strings in Python, try using ''.join([folder_name, '/*']) instead. You can read more about it here (Python's official recommendation):  https://www.python.org/dev/peps/pep-0008/#programming-recommendations

"For example, do not rely on CPython's efficient implementation of in-place string concatenation for statements in the form a += b or a = a + b. This optimization is fragile even in CPython (it only works for some types) and isn't present at all in implementations that don't use refcounting. In performance sensitive parts of the library, the ''.join() form should be used instead. This will ensure that concatenation occurs in linear time across various implementations."